src/logic/BrazeMessages.tsx

```diff
@@ -41,6 +41,7 @@ class BrazeMessage {
         this.slotName = slotName;
         this.cache = cache;
         this.errorHandler = errorHandler;
+        this.cache.errorHandler = errorHandler;
```

Comment:
Totally happy to go with your judgement here but I do think setting this on the platform before passing the cache in will give us more flexibility. For example if we decided we wanted to turn off the error tracking for the cache (let's say it turns out to be really noisy) then that would be a quick change on the platform(s), whereas this way you'd need to change the package, release a new version etc.